app/src/main/java/com/example/una/DummyFragment.java

```diff
@@ -0,0 +1,108 @@
+package com.example.una;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.example.una.models.Category;
+import com.loopj.android.http.AsyncHttpClient;
+import com.loopj.android.http.JsonHttpResponseHandler;
+import com.loopj.android.http.RequestParams;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+import cz.msebera.android.httpclient.Header;
+
+public class DummyFragment extends Fragment {
```

Comment:
You don't need a dummy fragment. You can use Denize's Home fragment

Reply:
I created a new Search Fragment (we were thinking of having a horizontal recycler view for categories that the user can use to search for charities, like the category pills you showed me on the Google Play Store) but I think my latest commit didn't show up here since I forgot to do `git rebase --continue`...